test/integration/puppetlabs/testutils.clj

```diff
@@ -0,0 +1,133 @@
+(ns puppetlabs.testutils
```

Comment:
This namespace might be one touch too generic :)  Maybe `puppetlabs.puppetserver.testutils`?  Since this code will end up in our test jar and be accessible to other projects, seems like it might be best to make sure the namespace at least includes the info about where it came from.